### PR TITLE
Updated WorldEdit for Sponge to 1.10.2

### PR DESCRIPTION
**Pull request by me4502**

### CODE REVIEW

File: worldedit-sponge/build.gradle [hunk 1]
```diff
@@ -35,8 +35,8 @@ sponge {
 }
 
 minecraft {
-    version = "1.8.9"
-    mappings = "stable_22"
+    version = "1.10.2"
+    mappings = "snapshot_20160629"
```
Comment (kenzierocks) [inline worldedit-sponge/build.gradle]: Any reason for this really old snapshot?

Comment (me4502) [inline worldedit-sponge/build.gradle]: I copied it from the forge version. thought it'd make sense to keep parity between the two.

Comment (kenzierocks) [inline worldedit-sponge/build.gradle]: I think it'd be better to track Sponge's mappings or latest.

**Comment by DarkArc** [comment]
Most if not all of these changes have been done for months in https://github.com/sk89q/WorldEdit/commits/feature/sponge-1.10.2 :sweat_smile:

**Comment by me4502** [comment]
D: why wasn't it merged?

**Comment by DarkArc** [comment]
Uh... Because 1.10 was still new, and you could get it on enginehub from the branch builds lol

**Comment by me4502** [comment]
aww :(

**Comment by DarkArc** [comment]
Could still be a change since then, might just want to rebase ontop of that, and see if anything got changed since I last updated that branch. I've been busy and haven't been doing much Minecraft stuff.

**Comment by DarkArc** [comment]
:+1: for initiative though :P

**Pull request closed without merging**